### PR TITLE
Finishing init rails_admin gem

### PR DESCRIPTION
**Pull request by binhthanhnguyen**

### CODE REVIEW

**Comment by minhld1709** [comment]
OK

**Comment by lannp** [comment]
OK

**Comment by hienbx94** [comment]
OK

**Comment by vietanhbkaaa** [comment]
OK

File: config/routes.rb [hunk 1]
```diff
@@ -1,11 +1,7 @@
 Rails.application.routes.draw do
+  mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
```
Comment (dieunb) [inline config/routes.rb]: double quote mark

**Pull request merged**